### ISSUE TITLE
GameCanvas.flushGraphics not implemented correctly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Perundurai-Products/microemu#6]
```
Now, in MicroEmulator, a call to flushGraphics results in a call to
repaint, which is not correct.

This causes many applications (esp. games) to flicker badly.

A call to flushGraphics should put the (already drawn) backbuffer on
screen.
Or, as SonyEricsson describes it:
"A call to flushGraphics() is similar to calling repaint(); and
serviceRepaints(); in sequence, but the difference is that flushGraphics()
does not generate a call to your paint() method."
```

Original issue reported on code.google.com by `bar...@gmail.com` on 18 Nov 2009 at 11:10

**Comment by GoogleCodeExporter** [comment]
Original comment by `bar...@gmail.com` on 6 Jan 2010 at 10:18
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in trunk (future 3.0 version)
```

Original comment by `bar...@gmail.com` on 17 Jan 2010 at 10:49
- Changed state: **Fixed**
- Added labels: **Milestone-3.0**
